src/mcpadapt/smolagents_adapter.py

```diff
@@ -85,7 +88,7 @@ def __init__(
                 self.is_initialized = True
                 self.skip_forward_signature_validation = True
 
-            def forward(self, *args, **kwargs) -> str:
+            def forward(self, *args, **kwargs):
```

Comment:
maybe we could type the return type here as image, audio or text

Reply:
Since `Pillow` and `torchaudio` are optional, I'm not sure that it won't throw error if the packages are not available